components/brave_vpn/common/brave_vpn_utils.cc

```diff
@@ -92,6 +92,7 @@ std::string GetBraveVPNEntryName(version_info::Channel channel) {
   switch (channel) {
     case version_info::Channel::CANARY:
       return entry_name + "-Nightly";
+    case version_info::Channel::UNKNOWN:
```

Comment:
AFAIK, `Channel::UNKNOWN` is used for unofficial build.
I think it's different with `Channel::DEV`.

Reply:
my local builds have unknown channel

Reply:
right - local build(non release) is unofficial and it'll have `UNKNOWN`.
Curious why `UNKNOWN` should use `-Dev` suffix?
It's for `dev` release channel now.

Reply:
If you want to suffix for unofficial build, I think `-Development` would be better.

Reply:
done